h/static/scripts/tests/controllers/form-controller-test.js

```diff
@@ -229,12 +229,19 @@ describe('FormController', () => {
   });
 
   it('displays an error if form submission fails without returning a new form', () => {
-    fakeSubmitForm.returns(Promise.reject({status: 500, reason: 'Internal Server Error'}));
+    fakeSubmitForm.returns(Promise.reject({status: 501, reason: 'Internal Server Error'}));
     return submitForm().then(() => {
       assert.equal(submitError(), 'Internal Server Error');
     });
   });
 
+  it('Displays a generic error message when an internal error occurs inside of submit form', () => {
+    fakeSubmitForm.returns(Promise.reject('An internal issue'));
```

Comment:
No change needed in this context but I'll just note for future that we try to follow the convention that promises are always rejected with Error-derived objects or Error-like objects (so regardless of what went wrong, you can access the error via the `message` property and in most cases, find the original via the `stack` property).

I did find this mentioned in a useful guide to Promise conventions somewhere but I couldn't find it just now.